### DISCUSSION TITLE
Selective Package Installation for Different Platforms in React Native

### DISCUSSION THREAD

**Discussion by rohan3342** [react-native-tvos/react-native-tvos#750] [Q&A]
### Description

We are facing an issue with package management across different platforms in our React Native project. Specifically, we need to install certain packages for TVOS and Android TV, but we do not want these packages to be installed for Android and iOS.

For example:
We want to use react-native-video for Android and iOS.
We want to use bitmovin-player-react-native for tvOS and Android TV.

Could you please provide guidance on how to configure our project to install specific packages only for TVOS and Android TV? Is there a recommended approach or best practice to achieve this selective package installation?

### Steps to reproduce

N/A

### React Native Version

0.69.0

### Affected Platforms

Runtime - Android, Runtime - iOS

### Output of `npx react-native info`

```text
N/A
```

### Stacktrace or Logs

```text
N/A
```

### Reproducer

N/A

**Comment by douglowder** [comment]
@rohan3342 I have converted your issue to a Q&A in our discussions section, and I am looking at some different ideas for how to resolve this. Expo projects have an easy way to handle this, but it appears that your project does not use Expo.

**Comment by freeboub** [comment]
On Android , if your manifest have only leanback launcher activity, play store will filter out phones and tablets !

**Reply by douglowder** [reply]
@freeboub I believe this is discussed here: https://github.com/react-native-tvos/config-tv/pull/14 and here: https://github.com/react-native-tvos/config-tv/issues/13

**Reply by freeboub** [reply]
I am not sure it will be clear enough...

Another solution for android can be to put: 
"android.software.leanback" as android:required="true".
